src/diracx/routers/auth.py

```diff
@@ -847,6 +847,14 @@ async def token(
     )
 
 
+@router.get("/refresh-tokens")
+async def refreshtokens(
+    auth_db: AuthDB,
+) -> list:
+    refreshtokens: list = await auth_db.get_refresh_tokens()
```

Comment:
>  - [ ] Add a route for listing a given users refresh tokens

Should a simple user get access to all the refresh tokens present in the database?

What about:
- a simple user gets only access to his/her (non-expired?) refresh tokens
- an admin user gets access to all the (non-expired?) refresh tokens
- a `user` parameter can be passed to `/refresh-tokens`:
  - it would return an error to a simple user trying to access the refresh tokens of another user
  - it would return the refresh tokens of the given user to an admin user. 

@bertrandrigaud any opinion?

Reply:
Yes of course we need to restrict information depending on user permissions. If we have a relative short retention for expired refresh tokens, I think we could also list it.